extensions/http-curl/processors/InvokeHTTP.cpp

```diff
@@ -115,6 +115,8 @@ void InvokeHTTP::setupMembersFromProperties(const core::ProcessContext& context)
 
   use_chunked_encoding_ = (context.getProperty(UseChunkedEncoding) | utils::andThen(&utils::StringUtils::toBool)).value_or(false);
   send_date_header_ = context.getProperty<bool>(DateHeader).value_or(true);
+  context.getProperty(UploadSpeedLimit, maximum_upload_speed_);
+  context.getProperty(DownloadSpeedLimit, maximum_download_speed_);
```

Comment:
These will parse the the properties as numbers, ignoring the suffix, so `Download Speed Limit = 10 MB/s` will result in `maximum_download_speed_ = 10` instead of `maximum_download_speed_ = 10000000`.

If you changed the type of these two fields to `core::DataTransferSpeedValue`, then it would work.  In that case, we would need to call `getValue()` when passing the values to curl.

Reply:
Good catch, updated in 41364556e36680ca7f8d564ffa08fe110188a34c